UnoCheck/CheckCommand.cs

```diff
@@ -367,6 +368,48 @@ private void CheckupStatusUpdated(object sender, CheckupStatusEventArgs e)
 			AnsiConsole.MarkupLine("  " + msg);
 		}
 
+        private string[] ParseTfmsToTargetPlatforms(CheckSettings settings)
+        {
+            var targetPlatforms = new List<string>();
+            foreach (var tfm in settings.Frameworks!)
+            {
+                var parsedTfm = NuGetFramework.ParseFolder(tfm);
+
+                if (parsedTfm.Version.Major >= 5 && parsedTfm.HasPlatform == false)
+                {
+                    // Returning empty list which means that we will target all platforms.
+                    return [];
+                } 
+                if (parsedTfm.HasPlatform)
+                {
+                    switch (parsedTfm.Platform)
+                    {
+                        case "windows":
+                            targetPlatforms.Add("windows");
+                            break;
+                        case "desktop":
+                            targetPlatforms.Add("win32");
```

Comment:
Desktop is actually windows, linux and mac at the same time. We may need to make its meaning based on the current platform running uno.check.

Reply:
Does it look better now?